lib/route/link.c

```diff
@@ -1565,6 +1585,9 @@ int rtnl_link_fill_info(struct nl_msg *msg, struct rtnl_link *link)
 	if (link->ce_mask & LINK_ATTR_NS_PID)
 		NLA_PUT_U32(msg, IFLA_NET_NS_PID, link->l_ns_pid);
 
+	if (link->ce_mask & LINK_ATTR_PERMANENT_ADDR)
+		NLA_PUT_ADDR(msg, IFLA_PERM_ADDRESS, link->l_paddr);
```

Comment:
does this make sense? As there is no `rtnl_link_set_perm_addr()`, is this still useful?

Maybe add a `rtnl_link_set_perm_addr()`, or just remove these two lines. Or why not?

Reply:
doesn't make sense and is removed now